app/javascript/packages/document-capture/components/unknown-error.tsx

```diff
@@ -13,6 +14,21 @@ interface UnknownErrorProps extends ComponentProps<'p'> {
   hasDismissed: boolean;
 }
 
+function formattedIdTypeMsg({ altFailedDocTypeMsg, acceptedIdUrl }) {
```

Comment:
I'd prefer naming like this `formatIdTypeMessage`
- No abbreviation
- Functions are usually verbs

Reply:
fixed.